packages/labs/analyzer/src/lib/analyzer.ts

```diff
@@ -81,10 +81,7 @@ export class Analyzer {
   };
 
   private _isLitElementModule = (file: ts.SourceFile) => {
-    return (
-      file.fileName.startsWith(this.packageRoot) &&
-      file.fileName.endsWith('/node_modules/lit-element/lit-element.d.ts')
-    );
+    return file.fileName.endsWith('/node_modules/lit-element/lit-element.d.ts');
```

Comment:
just curious, did something come up that prompted this change?

Reply:
Yeah: `lit-element` got hoisted out of the test-project when I made the test project a devDep of analyzer. We probably want to support hoisted installs for monorepos and things like pnpm anyway.